commons/src/main/scala/dagr/commons/io/Io.scala

```diff
@@ -143,4 +143,13 @@ object Io {
       case parent => findFirstExtentParent(parent)
     }
   }
+
+  /**
+  * Writes one or more lines to a file represented by a path.
+  */
+  def writeLines(path: Path, lines: Seq[String]) = {
+    val writer = Io.toWriter(path)
+    lines.foreach(line => writer.append(line).append('\n'))
+    writer.close()
```

Comment:
Agreed.  Perhaps if we write more of these we should create a `Destination` class with similar methods?